scout/server/config.py

```diff
@@ -39,7 +39,7 @@
 # )
 
 # Chanjo database connection string - used by chanjo report to create coverage reports
-# SQLALCHEMY_DATABASE_URI = "mysql+pymysql://test_user:test_passwordw@127.0.0.1:3306/chanjo"
+SQLALCHEMY_DATABASE_URI = "mysql+pymysql://test_user:test_passwordw@127.0.0.1:3306/chanjo"
```

Comment:
Uncommenting this line will create links to the report but when you click on them the app will crash. But perhaps we don't care?

Reply:
I think we will want to keep it commented out in the config example - this is just for testing!

Reply:
The intention is that we should just as before get no links if the chanjo module could not load, or if there is no `SQLALCHEMY_DATABASE_URI` configured. In case of config time errors, like db connection or password issues, we would still get a report time crash. We could move such errors up earlier by doing like a health check on the connection directly after init, but maybe its a bit overkill?

Reply:
overkill, yes!